types/part_set.go

```diff
@@ -155,6 +155,9 @@ type PartSet struct {
 	parts         []*Part
 	partsBitArray *bits.BitArray
 	count         uint32
+	// a count of the total size (in bytes). Used to ensure that the
+	// part set doesn't exceed the maximum block bytes
+	byteSize int64
```

Comment:
why int64?

Reply:
it matches MaxBytes in the Block Params which is also int64. tbh it's still a relatively arbitrary number